### PR TITLE
City page

### PR DESCRIPTION
**Pull request by xuejingao**

### CODE REVIEW

**Comment by rvente** [comment]
Fixes #3

**Comment by rvente** [comment]
Fixes #4

File: src/search_api/index.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const SearchForZipcode = async (zipcode) => {
  * @returns {Array<APIStateCityList>} All State and their cities found by city name
  */
 export const SearchCityName = async (cityName) => {
-  const zipcodes = await axios.get(`http://ctp-zip-api.herokuapp.com/city/${cityName.toUpperCase()}`);
+  const zipcodes = await axios.get(`https://ctp-zip-api.herokuapp.com/city/${cityName.toUpperCase()}`);
```
Comment (rvente) [inline src/search_api/index.jsx:48]: This was preventing our live version from working. Must have been something to do with mixed content. Just thought I'd fix it.

Comment (muskankapoor) [reply src/search_api/index.jsx:48]: Hey, @rvente and @xuejingao took a look at all your code and changes. it looks good to me. 
all your fixes look good.
all working well from your end, please merge the pull request when you get a chance.

**Review by muskankapoor — APPROVED** [review]

**Pull request merged**